src/test/java/org/minions/devfund/damianvp/moviedatabase/MovieDataBaseTest.java

```diff
@@ -0,0 +1,172 @@
+package org.minions.devfund.damianvp.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.ArrayList;
+import static org.junit.Assert.assertEquals;
+
+/**
+ *  Class to test movie database.
+ */
+public class MovieDataBaseTest {
+    private MovieDatabase movieDatabase;
+    private String actorMacchio = "Ralph Macchio";
+    private String actorFreeman = "Morgan Freeman";
+    private String actorBale = "Christian Bale";
+    private String actorJolie = "Angelina Jolie";
+    private String karateMovie = "Karate Kid";
+    private String gloryMovie = "Glory";
+    private String sevenMovie = "Seven";
+    private String[] actorList1 = {actorMacchio, actorFreeman};
+    private String[] actorList2 = {actorMacchio, actorBale, actorJolie};
+    private String[] actorList3 = {actorJolie, actorFreeman};
```

Comment:
Don't declare this variables here
Please declare them as final local variables inside test method.